bin/addClassToBaseline.py

```diff
@@ -74,7 +74,8 @@ def main():
 
     provenance = extractor.getProvenance(imagePath, maskPath, mask)
 
-    image, mask, bb = imageoperations.cropToTumorMask(image, mask)
```

Comment:
> Sorry, I understand this is not related to the essence of this specific PR

No, it's not, but it's still a good suggestion. I am running over the documentation for errors, will update this then also.